docs/src/user-guide/core-unstructured/clp.md

```
@@ -105,24 +105,32 @@
   records.
-* Copy and change `config/metadata-db.yml`, setting the type to `mysql` and uncommenting the MySQL
-  parameters.
-* Install the MariaDB and PyYAML Python packages `pip3 install mariadb PyYAML`
+* Install the MariaDB Python package `pip3 install mariadb`
   * This is necessary to run the database initialization script. If you prefer, you can run the SQL
     statements in `tools/scripts/db/init-db.py` directly.
-* Run `tools/scripts/db/init-db.py` with the updated config file. This will initialize the database
-  CLP requires.
-* Run `clp` or `clg` as before, with the addition of the `--db-config-file` option pointing at the
-  updated config file.
+* Run `tools/scripts/db/init-db.py`. This will initialize the database CLP requires. Use 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)